sphinx/source/docs/user_guide/source_examples/styling_arrow_annotations.py

```diff
@@ -13,6 +13,6 @@
 
 for i, style in enumerate(ARROW_HEADS):
     p.add_layout(Arrow(x_start=0.2, y_start=i, x_end=0.2, y_end=i, end=style()))
-    p.add_layout(Label(x=0.2, x_offset=20, y=i, text=[style.__name__], text_baseline='middle', text_align='left'))
+    p.add_layout(Label(x=0.2, x_offset=20, y=i, text=style.__name__, text_baseline='middle', text_align='left'))
```

Comment:
I also added this small docs example fix. Arrow `text` is no longer a `DataSpec`